src/buildtool.unit.test.ts

```diff
@@ -10,7 +11,7 @@ describe("command generation", () => {
         };
 
         const actual = buildtool.generateCommand(options);
-        expect(actual).toBe("buildtool")
+        expect(actual).toBe("addpath('"+ path.join(__dirname, "plugins").replace("'","''") +"');buildtool")
```

Comment:
Could we set this to be a variable or a function so that we don't need to repeat thius logic in every test case?

Reply:
Done.